lib/sendgrid/helpers.rb

```diff
@@ -0,0 +1,20 @@
+module SendGrid
+  module Helpers
+    def to_json
```

Comment:
I'm not sure about doing this magically.  This seems like a landmine.  I can see someone, unaware of the convention this establishes, adding a new accessor that is then unwittingly serialized.